wolfcrypt/test/test.c

```diff
@@ -3141,9 +3147,17 @@ WOLFSSL_TEST_SUBROUTINE wc_test_ret_t sha256_test(void)
     byte      hashcopy[WC_SHA256_DIGEST_SIZE];
     wc_test_ret_t ret = 0;
 
-    testVector a, b, c;
-    testVector test_sha[3];
+    testVector a, b, c, d;
+    testVector test_sha[4];
+#ifndef NO_WOLFSSL_SHA256_INTERLEAVE
+    byte       i_hash[WC_SHA256_DIGEST_SIZE];
+    byte       i_hashcopy[WC_SHA256_DIGEST_SIZE];
+    testVector interleave_test_sha[4];
+    wc_Sha256  i_sha, i_shaCopy;
+#endif
+
     int times = sizeof(test_sha) / sizeof(struct testVector), i;
+    active_wolfssl_test = "sha256_test";
```

Comment:
If you are adding `active_wolfssl_test` please expand or provide a way to eliminate for small targets.

Reply:
I've removed the `active_wolfssl_test` after discussing with @bandi13. Same functionality, but much better. 

See revised PR description.